src/TouristInfoCards.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+
+const TouristInfoCards = () => {
```

Comment:
This component is responsible for rendering all of the Tourist Info cards.

Could instead make a component that is responsible for rendering a single card? What props would you pass to this component?

Reply:
I should make an array of objects to store those data and pass it as prop to TouristInfoCards and loop through it to render by using only a single card.